### DISCUSSION TITLE
Move JS-object inline?

### DISCUSSION THREAD

**Discussion by FredMar** [alpinejs/alpine#3531] [1. Help] [answered]
Hi,

Is there a way to move this JS-object inline? If I try do do it, it will throw and error. It would be great to get rid of the script tags.

![image](https://user-images.githubusercontent.com/16269671/232482570-6cabb0df-03ae-4ec3-bad4-457aaed7329d.png)

Thanks :)

**Comment by ekwoka** [comment, marked as answer]
Yeah, just put it in there. Technically, it doesn't even need to be inside JSON.parse since the JSON is also a valid object literal in javascript.

oh right, the `"` breaks the html attributes.

Yeah, not much you can do about that.

**Reply by FredMar** [reply]
Thanks for your time!
